keras_contrib/applications/densenet.py

```diff
@@ -275,7 +419,7 @@ def __create_dense_net(nb_classes, img_input, include_top, depth=40, nb_dense_bl
         dropout_rate: dropout rate
         weight_decay: weight decay
 
-    Returns: keras tensor with nb_layers of __conv_block appended
+    Returns: keras tensor with nb_layers of conv_block appended
```

Comment:
No. I use __ in all the methods to make them hidden to the end user. However, when referring to code blocks, there is no need to mention it in private form.